### PR TITLE
Seeker renamed to MobileUnit

### LINKED ISSUE #295: Rename code variables to match latest game language
For any code that is part of the high level code docs, change any variable names to match the names in latest game language.

1. Create a list of suggested changes (e.g. 
2. Get that approved by @lukgib from game language side
3. Get that approved from all code team from a variable team side
4. Agree a good time to do the switch with code team to avoid annoying conflicts.
5. Do it (This step might be done by someone else)

example change proposal

| Game Language Term | Code File Defintion     | Existing  | Proposed|
| ----------- | ----------- |---|--|
| Unit      | contracts/src/schema/Schema.sol      |  Kind.Seeker |  Kind.MobileUnit |

### PR DESCRIPTION
**Pull request by 5p0rt5BEArD**
All occurrences of "seeker" have been renamed to "mobileUnit" preserving the correct casing. This includes variable names, string identifiers and filenames.
 
All references to DawnSeekers have been renamed to Downstream apart from the DawnSeekersUnity folder which is now 'map'

⚠️ With this pushed,  ds-hammer-factory and ds-example need updating

PRs to do this:
https://github.com/playmint/ds-hammer-factory/pull/14
https://github.com/playmint/ds-examples

All individual building code will also need updating.

fixes #295

### CODE REVIEW

File: bridge/README.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ This will build to `bridge/build`. This is the path used by `PluginController.cs
 
 ## Output
 
-Dawnseekers bridge handles the state update events from the Dawnseekers client, removes circular references from the object, serialises the state as JSON and outputs it on stdout which is read by Unity. A replacer function is used to replace the BigInts in the original state object with hex strings.
+Downstream bridge handles the state update events from the Dawnseekers client, removes circular references from the object, serialises the state as JSON and outputs it on stdout which is read by Unity. A replacer function is used to replace the BigInts in the original state object with hex strings.
```
Comment (chrisfarms) [inline bridge/README.md:26]:
```suggestion
Downstream bridge handles the state update events from the Downstream client, removes circular references from the object, serialises the state as JSON and outputs it on stdout which is read by Unity. A replacer function is used to replace the BigInts in the original state object with hex strings.
```

File: core/src/mobileUnit.ts
Comment (chrisfarms) [inline core/src/mobileUnit.ts:1]: naming convention for files in js land is all lowercase with hyphens ... but we can tidy up later

File: frontend/src/components/views/shell/shell.styles.ts [hunk 2]
```diff
@@ -117,8 +117,8 @@ const baseStyles = (_: Partial<ShellProps>) => css`
         }
     }
 
-    .seeker-actions {
-        .seeker-selector {
+    .mobileUnit-actions {
```
Comment (chrisfarms) [inline frontend/src/components/views/shell/shell.styles.ts:120]: should be lowercase + hyphen in css land ... but it won't cause issues

**Review by chrisfarms — CHANGES_REQUESTED** [review]
👏 well done

❌ I'm unable to get a local build working from this branch, looks like something todo with addressables, so blocking til I've worked out what is going on

**Comment by chrisfarms** [comment]
It looks like all the addressable asset settings have been deleted rather than moved .... `map/Assets/AddressableAssetsData/AddressableAssetSettings.asset` is missing for instance.

I suspect this is a result of a `.gitignore` rule on a directory that had some files explicitly commited to it .... so during rename of the directory it may have hid the change from you.

you may want to check if there are any other cases like this.... or maybe use `git mv` instead of a standard `mv`, `git mv` is usually smart enough to deal with this kind of thing

File: core/src/mobileUnit.ts
Comment (5p0rt5BEArD) [reply core/src/mobileUnit.ts:1]: will fix it now
and .graphql files? So mobileUnit.graphql should be mobile-unit.graphql ?

File: frontend/src/components/views/shell/shell.styles.ts (hunk 2 above)
Comment (5p0rt5BEArD) [reply frontend/src/components/views/shell/shell.styles.ts]: fixed

File: core/src/mobileUnit.ts
Comment (5p0rt5BEArD) [reply core/src/mobileUnit.ts:1]: i've made it mobile-unit.graphql

**Comment by 5p0rt5BEArD** [comment]
> 

/Assets/AddressableAssetsData/ is the .gitignore string so this shouldn't change. I think they get generated on build so not sire why they're not for you. Perhaps its an editor only generation - I'm investigating that.

I did find that I'd missed a folder rename in the .dockerignore - maybe that was preventing your build?

**Comment by chrisfarms** [comment]
> > 
> 
> /Assets/AddressableAssetsData/ is the .gitignore string so this shouldn't change. I think they get generated on build so not sire why they're not for you. Perhaps its an editor only generation - I'm investigating that.
> 
> I did find that I'd missed a folder rename in the .dockerignore - maybe that was preventing your build?

**Files can still be explicitly tracked, even if everything else under the folder is being ignored.**

The following files were previously being tracked and present in the repo under DawnSeekersUnity/Assets/AddressableAssetsData, but have been DELETED in your branch (which is probably why you have a negative total change count even tho you renamed things to a longer name!)

```bash
DawnSeekersUnity/Assets/AddressableAssetsData [main]$ git ls-tree -r HEAD --name-only .

AddressableAssetSettings.asset
AddressableAssetSettings.asset.meta
AssetGroupTemplates.meta
AssetGroupTemplates/Packed Assets.asset
AssetGroupTemplates/Packed Assets.asset.meta
AssetGroups.meta
AssetGroups/AccountDemo.asset
AssetGroups/AccountDemo.asset.meta
AssetGroups/Built In Data.asset
AssetGroups/Built In Data.asset.meta
AssetGroups/Default Local Group.asset
AssetGroups/Default Local Group.asset.meta
AssetGroups/Schemas.meta
AssetGroups/Schemas/AccountDemo_BundledAssetGroupSchema.asset
AssetGroups/Schemas/AccountDemo_BundledAssetGroupSchema.asset.meta
AssetGroups/Schemas/AccountDemo_ContentUpdateGroupSchema.asset
AssetGroups/Schemas/AccountDemo_ContentUpdateGroupSchema.asset.meta
AssetGroups/Schemas/Built In Data_PlayerDataGroupSchema.asset
AssetGroups/Schemas/Built In Data_PlayerDataGroupSchema.asset.meta
AssetGroups/Schemas/Default Local Group_BundledAssetGroupSchema.asset
AssetGroups/Schemas/Default Local Group_BundledAssetGroupSchema.asset.meta
AssetGroups/Schemas/Default Local Group_ContentUpdateGroupSchema.asset
AssetGroups/Schemas/Default Local Group_ContentUpdateGroupSchema.asset.meta
AssetGroups/Schemas/System_BundledAssetGroupSchema.asset
AssetGroups/Schemas/System_BundledAssetGroupSchema.asset.meta
AssetGroups/Schemas/System_ContentUpdateGroupSchema.asset
AssetGroups/Schemas/System_ContentUpdateGroupSchema.asset.meta
AssetGroups/System.asset
AssetGroups/System.asset.meta
DataBuilders.meta
DataBuilders/BuildScriptFastMode.asset
DataBuilders/BuildScriptFastMode.asset.meta
DataBuilders/BuildScriptPackedMode.asset
DataBuilders/BuildScriptPackedMode.asset.meta
DataBuilders/BuildScriptPackedPlayMode.asset
DataBuilders/BuildScriptPackedPlayMode.asset.meta
DataBuilders/BuildScriptVirtualMode.asset
DataBuilders/BuildScriptVirtualMode.asset.meta
DefaultObject.asset
DefaultObject.asset.meta
OSX.meta
OSX/addressables_content_state.bin
OSX/addressables_content_state.bin.meta
ProfileDataSourceSettings.asset
ProfileDataSourceSettings.asset.meta
WebGL.meta
WebGL/addressables_content_state.bin.meta
```

it's possible some of those were being incorrectly tracked ... but I **know** that at the very least the Asset settings and AssetGroup stuff must be present as this contains the config for how to build the streaming asset bits and where to put it

**Comment by 5p0rt5BEArD** [comment]
> **Files can still be explicitly tracked, even if everything else under the folder is being ignored.**

Right. Those are the files I managed to delete then. Rename folder above them, lost their explicit tracking and they fell under the .gitignore rule so didn't get re-added.

I've redone the rename with a git-mv and  I think its all good now.

Lesson learned!

**Review by chrisfarms — APPROVED** [review]

**Review by ldunnplaymint — APPROVED** [review]

**Pull request merged**